index.html

```diff
@@ -350,14 +347,14 @@ <h2 class="footer-text">
               name="user_message"
               maxlength="500"
               required
+              id="usertext"
```

Comment:
- [ ] <b>REQUIRED:</b> The `Id` you are using here is not the one you used in JavaScript, so it still gonna throw an error saying `userText is null`. to fix the issue, Kindly use `user-text` instead of  `usertext`. or simply change the one in JavaScript.